v0/sources/satismeter/utils.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
+const { isDefinedAndNotNullAndNotEmpty } = require("../../util");
+
+/**
+ * This function removes all those variables which are
+ * empty or undefined or null from all levels of object.
+ * @param {*} obj
+ * @returns payload without empty null or undefined variables
+ */
+const refinePayload = obj => {
```

Comment:
Is there any scope of making this available in root-level utils, so that it can be reusable?

(Non blocking)

Reply:
Yeah absolutely we can do it .. but I guess there were changes going on regarding the is isdefinednotnullandnotempty one. Will reach out respective stakeholders and will do the appropriate.